.github/workflows/harvey.yml

```diff
@@ -16,6 +16,16 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y rc qemu go-dep qemu-utils expect qemu-system-x86
         go get harvey-os.org/cmd/ufs
+    - name: install source we build into the image
+      run: |
+        git clone --depth 1 --branch go1.16.6 https://github.com/golang/go.git
+        export GOBIN=`pwd`/amd64/go/bin
+        echo $GOBIN
+        mkdir -p $GOBIN
+        ls -R $GOBIN
+        cd go/src
+        GOOS=plan9 GOARCH=amd64 ./make.bash
+        ls -R $GOBIN
```

Comment:
To verify, a little `file $GOPATH/...` could do.

Reply:
the file command seems to view plan 9 binaries as 'data'